backend/doc/apache.md

```diff
@@ -1,9 +1,9 @@
-# Running Tracim through Apache #
+# Using Apache as a proxy for Tracim #
```

Comment:
`s/as a proxy/as a Proxy/` ?

Reply:
I would keep proxy as is, I think it is a [regular name](https://en.wiktionary.org/wiki/proxy#Noun) and not a product name.